pyfr/plugins/base.py

```diff
@@ -62,6 +62,25 @@ def serialise(self, intg):
         return {}
 
 
+class CLIMixin:
```

Comment:
I would not use a mixin here.  It causes issues since we can never construct what it is mixed in with (as that requires an integrator).  Thus, I'd go for an entirely separate class hierarchy.